Do not install kill and uptime, use /lib for all archs.

--- Makefile.orig	2006-06-24 11:02:25.000000000 +0200
+++ Makefile	2008-10-24 08:40:57.000000000 +0200
@@ -39,20 +39,20 @@ usr/proc/bin             := $(DESTDIR)/u
 man1                     := $(DESTDIR)/usr/share/man/man1/
 man5                     := $(DESTDIR)/usr/share/man/man5/
 man8                     := $(DESTDIR)/usr/share/man/man8/
-lib                      := $(DESTDIR)/$(lib64)/
-usr/lib                  := $(DESTDIR)/usr/$(lib64)/
+lib                      := $(DESTDIR)/lib/
+usr/lib                  := $(DESTDIR)/usr/lib/
 usr/include              := $(DESTDIR)/usr/include/
 
 #SKIP     := $(bin)kill $(man1)kill.1
 
-BINFILES := $(usr/bin)uptime $(usr/bin)tload $(usr/bin)free $(usr/bin)w \
+BINFILES := $(usr/bin)tload $(usr/bin)free $(usr/bin)w \
             $(usr/bin)top $(usr/bin)vmstat $(usr/bin)watch $(usr/bin)skill \
-            $(usr/bin)snice $(bin)kill $(sbin)sysctl $(usr/bin)pmap \
+            $(usr/bin)snice $(sbin)sysctl $(usr/bin)pmap \
             $(usr/proc/bin)pgrep $(usr/proc/bin)pkill $(usr/bin)slabtop \
             $(usr/proc/bin)pwdx
 
-MANFILES := $(man1)uptime.1 $(man1)tload.1 $(man1)free.1 $(man1)w.1 \
-            $(man1)top.1 $(man1)watch.1 $(man1)skill.1 $(man1)kill.1 \
+MANFILES := $(man1)tload.1 $(man1)free.1 $(man1)w.1 \
+            $(man1)top.1 $(man1)watch.1 $(man1)skill.1 \
             $(man1)snice.1 $(man1)pgrep.1 $(man1)pkill.1 $(man1)pmap.1 \
             $(man5)sysctl.conf.5 $(man8)vmstat.8 $(man8)sysctl.8 \
             $(man1)slabtop.1 $(man1)pwdx.1
